pom.xml

```diff
@@ -156,14 +156,11 @@
                     <execution>
                         <id>prepare-feature-launcher</id>
                         <goals>
-                            <goal>get</goal>
-                            <goal>copy</goal>
+                            <goal>unpack</goal>
                         </goals>
                         <phase>package</phase>
                         <configuration>
-                            <!-- 1.1.28 and newer versions don't work out-of-the-box due to SLING-10956 -->
-                            <artifact>org.apache.sling:org.apache.sling.feature.launcher:1.1.26</artifact>
-                            <stripVersion>true</stripVersion>
+                            <artifact>org.apache.sling:org.apache.sling.feature.launcher:1.2.0:tar.gz</artifact>
```

Comment:
Why did you remove the `stripVersion` config? With it configured we don't have to update the documentation every time we update the feature launcher version.

Reply:
> Why did you remove the `stripVersion` config?

Mainly because the stripVersion config doesn't appear to be supported by the unpack goal.  That config was for renaming the artifact.  The version number you are seeing is the folder name from inside the .tar.gz file that was unpacked.  

I'll try to use the <fileMappers> config to rewrite the target path of that folder to achieve the same result.